automatic/sublimetext4/tools/chocolateyUninstall.ps1

```diff
@@ -1,26 +1,3 @@
 $ErrorActionPreference = 'Stop'
 
-$packageName = "sublimetext4"
-
-[array]$key = Get-UninstallRegistryKey -SoftwareName "Sublime Text 4"
```

Comment:
The code for uninstallation is required because `QuietUninstallString` contains `"C:\Program Files\Sublime Text\unins000.exe" /SILENT` which isn't enough for the uninstall to be fully silent - the uninstall window still pops up.

However, as you've noticed, Sublime Text 4's installer has been changed to Inno Setup which seems to have made this code do nothing due to the software name no longer changing from `Sublime Text 4` to `Sublime Text_is1` so this will have to be fixed.

Reply:
I would just put it as part of the repo.

Reply:
Fixed in as part of 8d0ef9cc9028704a261c285ebc21d30e21c4643f